File: android/src/main/kotlin/ws/dyt/plugin/androidapk/entity/AppInfo.kt
package ws.dyt.plugin.androidapk.entity

/**
 * Created by yangxiaowei
 */
data class AppInfo(
    var appName: String? = null,
    //bitmap base64
    var appIcon: String? = null,
    //px
    var appIconWidth: Int? = 0,
    //px
    var appIconHeight: Int? = 0,
    var packageName: String? = null,
    var versionCode: Int = 0,
    var versionName: String? = null
)

File: lib/plugin/market/AppInfo.dart
import 'dart:convert';

/// Created by yangxiaowei
class AppInfo {

    String appName;
    //bitmap base64
    String appIcon;
    //px
    int appIconWidth;
    //px
    int appIconHeight;
    String packageName;
    int versionCode;
    String versionName;

    AppInfo.fromMap(Map map) {
        this.appName = map['appName'];
        this.appIcon = map['appIcon'];
        this.appIconWidth = map['appIconWidth'];
        this.appIconHeight = map['appIconHeight'];
        this.packageName = map['packageName'];
        this.versionCode = map['versionCode'];
        this.versionName = map['versionName'];
    }

    static List<AppInfo> transfer(String gson) {
        List<dynamic> list = json.decode(gson);

        return list.map((item) {
            return new AppInfo.fromMap(item);
        }).toList();
    }
}